File: JavaScript/8js 2/multiply_five.js
/*
|--------------------------------------------------------------------------
| Multiply the number
|--------------------------------------------------------------------------
|
*/

function multiply(number) {
   return number * 5 ** String(Math.abs(number)).length;
}


function multiply(number) {
   return number * Math.pow(5, Math.abs(number).toString().length);
}


const multiply = (number) => number * Math.pow(5, (Math.abs(number) + "").length);


console.log(multiply(10), 250);
console.log(multiply(5), 25);
console.log(multiply(200), 25000);
console.log(multiply(0), 0);
console.log(multiply(-2), -10);

File: Python/8py 2/multiply_five.py
"""
|--------------------------------------------------------------------------
| Multiply the number
|--------------------------------------------------------------------------
|
"""

def multiply(n):
   return n * 5 ** len(str(abs(n)))


def multiply(n: int) -> int:
   return n * pow(5, len(str(abs(n))))


multiply=lambda n: n*5**len(str(abs(n)))


print(multiply(10),250)
print(multiply(5),25)
print(multiply(200),25000)
print(multiply(0),0)
print(multiply(-2),-10)